README.md

```diff
@@ -104,6 +104,8 @@ An `EventLoopFuture<T>` is essentially a container for the return value of a fun
 
 If you had to poll the future to detect when it completed that would be quite inefficient, so `EventLoopFuture<T>` is designed to have managed callbacks. Essentially, you can hang callbacks off the future that will be executed when a result is available. The `EventLoopFuture<T>` will even carefully arrange the scheduling to ensure that these callbacks always execute on the event loop that initially created the promise, which helps ensure that you don't need too much synchronization around `EventLoopFuture<T>` callbacks.
 
+Another important topic for consideration is the difference between the  `close()`  method and the `closeFuture: EventLoopFuture` property on a `Channel`.  For example, when the `close()` method is called from a `Channel` an I/O request is put into the `ChannelPipeline` to forward this request to the next `ChannelHandler` in the pipeline or complete the `promise: EventLoopPromise` on a `Channel`.  Completing this promise on the `Channel` should then fulfill the `closeFuture: EventLoopFuture` property and code can be executed based upon the  completion of this event.
+
```

Comment:
So this doesn't quite hit the theme I wanted this doc to cover. The specific difference is between the promise passed to `close` and the one that we're hanging off the `Channel` with the name `closeFuture`.

The specific semantics we want to get across is that the promise passed to `close` is fired after the specific channel is closed down, but before the pipeline is destroyed. If you want to wait for a channel to be completely resolved with no further possible work, wait for the `closeFuture` to succeed. If you want to take actions on the channel pipeline before it is cleared out, wait for the `EventLoopPromise` passed to `close` to succeed.